src/System.Reflection.TypeExtensions/src/System/Reflection/TypeExtensions.cs

```diff
@@ -177,25 +177,25 @@ public static PropertyInfo[] GetProperties(Type type, BindingFlags bindingAttr)
             return type.GetProperties(bindingAttr);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name)
+        public static PropertyInfo? GetProperty(Type type, string name)
         {
             Requires.NotNull(type, nameof(type));
             return type.GetProperty(name);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name, BindingFlags bindingAttr)
+        public static PropertyInfo? GetProperty(Type type, string name, BindingFlags bindingAttr)
         {
             Requires.NotNull(type, nameof(type));
             return type.GetProperty(name, bindingAttr);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name, Type returnType)
+        public static PropertyInfo? GetProperty(Type type, string name, Type returnType)
         {
             Requires.NotNull(type, nameof(type));
             return type.GetProperty(name, returnType);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name, Type returnType, Type[] types)
+        public static PropertyInfo? GetProperty(Type type, string name, Type returnType, Type[] types)
```

Comment:
GitHub won't let me comment on it, but Type.IsAssignableFrom accepts a `Type?`... we don't want to do the same with the below IsAssignableFrom?

Reply:
Same for IsInstanceOfType

Reply:
Ha, we commented at the same time 😄